main.go

```diff
@@ -21,13 +21,14 @@ import (
 // OutputFormatVersion will allow us to filter output files downstream.
 const OutputFormatVersion = '1'
 
-// CPUTimesStat is a more limited version of cpu.TimesStat to save storage space
-type CPUTimesStat struct {
-	User   float64 `json:"user"`
-	System float64 `json:"system"`
-	Idle   float64 `json:"idle"`
-	Iowait float64 `json:"iowait"`
-	Steal  float64 `json:"steal"`
+// ProcCPUStat is a more limited version of cpu.TimesStat to save storage space
+type ProcCPUStat struct {
```

Comment:
I renamed this to include the `Proc` prefix to be consistent with other structs collecting per/process measurements.

Reply:
We can rename them if it's not clear, it's no trouble